### DISCUSSION TITLE
Add support for safari and firefox backdrop blur

### DISCUSSION THREAD

**Discussion by snax4a** [tailwindlabs/tailwindcss#7257] [Ideas]
Hi,
Current tailwind backdrop-blur utility only works in chrome and edge.
To make it working on safari I need to manually add this css -webkit-backdrop-filter: blur(4px);

Firefox does not support backdrop filter but if safari does why don't you add -webkit properties to tailwindcss?

**Comment by adamwathan** [comment]
Hey! Adding vendor prefixes should be handled by autoprefixer as explained in the docs:

https://tailwindcss.com/docs/browser-support#vendor-prefixes

No way for us to make it work in Firefox of course, that would need to be implemented in the browser. Firefox does support it but you need to enable it behind an experimental flag in the browser preferences.

**Reply by snax4a** [reply]
> Hey! Adding vendor prefixes should be handled by autoprefixer as explained in the docs:
> 
> https://tailwindcss.com/docs/browser-support#vendor-prefixes
> 
> No way for us to make it work in Firefox of course, that would need to be implemented in the browser. Firefox does support it but you need to enable it behind an experimental flag in the browser preferences.

Thank you, I have added autoprefixer to my postcss config and its working
